packages/typography/README.md

```diff
@@ -176,6 +176,7 @@ All props extend the HTMLElementProps of their root tag, however the below compo
 | `hideExternalIcon` | `boolean`                        | Hides the external icon when the current host name is different from the host of the destination URL                                                                                     | `false`                                       |
 | `darkMode`         | `boolean`                        | Determines if the component renders in dark mode                                                                                                                                         | `false`                                       |
 | `as`               | `keyof JSX.IntrinsicElements`    | Element to render the component as. **Note**: This will supersede the behavior of any other props.                                                                                       | `'a'`                                         |
+| `inline`           | `boolean`                        | Determines if the Link should be inherit font size from the parent                                                                                                                       | `false`                                       |
```

Comment:
This feels like it's going to apply `display: inline`. Prop should be something like `inheritFontSize`